src/engine/value.rs

```diff
@@ -26,6 +26,33 @@ pub enum Value {
     UInt64(u64),
 }
 
+impl Value {
```

Comment:
Why not implement this as `impl TryFrom<Value> for String`?

Reply:
I think I tried that and I couldn't get it to work, probably because I was doing something wrong. I'll go back and try it again.

Reply:
Fixed.